Formula/rkflashtool.rb

```diff
@@ -16,15 +16,9 @@ class Rkflashtool < Formula
   end
 
   depends_on "libusb"
-
-  # Add file 'version.h' that has been forgotten in the tarball
-  resource "version" do
-    url "https://downloads.sourceforge.net/project/rkflashtool/rkflashtool-5.1/version.h"
-    sha256 "b7bd8e3c66170d5f6e6a68c5add677ad365e37b4121251de87ad29e7fdf06cd6"
-  end
+  depends_on "pkg-config" => :build
 
   def install
-    resource("version").stage { buildpath.install "version.h" } if build.stable?
     system "make"
```

Comment:
I've tried this line instead of existing ones, but I've got following error message:

```==> Downloading https://homebrew.bintray.com/bottles/rkflashtool-6.1.sierra.bottle.1.tar.gz

curl: (22) The requested URL returned error: 404 Not Found
Error: Failed to download resource "rkflashtool"
Download failed: https://homebrew.bintray.com/bottles/rkflashtool-6.1.sierra.bottle.1.tar.gz
Warning: Bottle installation failed: building from source.
==> Using the sandbox
==> Downloading https://downloads.sourceforge.net/project/rkflashtool/rkflashtool-6.1/rkflashtool-6.1-src.tar.bz2
Already downloaded: /Users/cleg/Library/Caches/Homebrew/rkflashtool-6.1.tar.bz2
==> make install
Last 15 lines from /Users/cleg/Library/Logs/Homebrew/rkflashtool/01.make:
2017-05-17 13:58:16 +0200

make
install

/bin/sh: pkg-config: command not found
gcc -O2 -W -Wall -Wextra -pedantic -std=c99 -D_XOPEN_SOURCE=600 -I/usr/include/libusb-1.0 rkcrc.c -o rkcrc  -lusb-1.0
gcc -O2 -W -Wall -Wextra -pedantic -std=c99 -D_XOPEN_SOURCE=600 -I/usr/include/libusb-1.0 rkflashtool.c -o rkflashtool  -lusb-1.0
gcc -O2 -W -Wall -Wextra -pedantic -std=c99 -D_XOPEN_SOURCE=600 -I/usr/include/libusb-1.0 rkunpack.c -o rkunpack  -lusb-1.0
rkflashtool.c:35:10: fatal error: 'libusb.h' file not found
#include <libusb.h>
         ^
1 error generated.
make: *** [rkflashtool] Error 1
make: *** Waiting for unfinished jobs....

READ THIS: http://docs.brew.sh/Troubleshooting.html```

And mine knowledge of toolchain isn't good enough to sort this out. I suppose I should specify prefix, so I've tried different variations like `system "make", "PREFIX=#{prefix}"`, etc, but it's not working too. So I've decided to close this pull request